curriculum/challenges/_meta/bootstrap/meta.json

```diff
@@ -7,6 +7,9 @@
   "required": [
     {
       "link": "https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.css"
+    },
+    {
+      "link": "https://use.fontawesome.com/releases/v5.5.0/css/all.css"
```

Comment:
IMO, it's not needed here. Only a few challenges require FA and they contain a link to it. Now both links from a `meta` file and from a challenge file are used in the client and for test challenges. Let's remove it from this PR and if you want you'll open new PR with updated links in challenge files.